src/radix_connect/mobile/session_storage.rs

```diff
@@ -0,0 +1,16 @@
+use crate::prelude::*;
+
+#[uniffi::export(with_foreign)]
+#[async_trait::async_trait]
+pub trait RadixConnectMobileSessionStorage: Send + Sync {
```

Comment:
missing doc, maybe a short "Raison d'être": ` /// Implemented by hosts so that Sargon can save RCfM sessions.`